### ISSUE TITLE
Priority order to focus

### ISSUE THREAD

**Issue by hiperesp** [hiperesp/DragonFable-Private-Server#21]
Priority order to focus:

- [x] cf-questcomplete-Mar2011.asp
- [x] cf-shopload.asp
- [x] shop: buy/sell action
- [x] cf-bankload.asp
- [ ] cf-questreward.asp

**Comment by hiperesp** [comment]
**Request to `cf-questcomplete-Mar2011.asp`**
```xml
<flash><intWaveCount>1</intWaveCount><intRare>0</intRare><intWar>0</intWar><intLootID>-1</intLootID><intExp>2565</intExp><intGold>122</intGold><intQuestID>16</intQuestID><strToken>TOKEN</strToken><intCharID>1</intCharID></flash>
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<questreward xmlns:sql="urn:schemas-microsoft-com:xml-sql"><questreward intExp="10590" intSilver="0" intGold="851" intGems="0" intCoins="3"><items ItemID="7" strItemName="Newbie Ring" strItemDescription="It feels as cheap as it looks. It is exactly what you expect for the level 1 warrior wanting to make a name for himself." bitVisible="1" bitDestroyable="1" bitSellable="1" bitDragonAmulet="0" intCurrency="2" intCost="20" intMaxStackSize="1" intBonus="0" intRarity="0" intLevel="1" strType="Melee" strElement="Metal" strCategory="Armor" strEquipSpot="Finger" strItemType="Ring" strFileName="" strIcon="ring" intStr="0" intDex="0" intInt="0" intLuk="0" intCha="0" intEnd="0" intMin="0" intMax="0" intDefMelee="1" intDefPierce="0" intDefMagic="1" intCrit="0" intParry="0" intDodge="0" intBlock="0" strResists=""/></questreward></questreward>
```

**Request to `cf-questreward.asp`**
```xml
<flash><intNewItemID>7</intNewItemID><strToken>TOKEN</strToken><intCharID>1</intCharID></flash>
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<questreward xmlns:sql="urn:schemas-microsoft-com:xml-sql"><CharItemID>783730404</CharItemID></questreward>
```

**Comment by hiperesp** [comment]
Closed due to new created task: #70
